tekton/release-pipeline.yaml

```diff
@@ -80,6 +99,17 @@ spec:
           subpath: git
     - name: unit-tests
       runAfter: [precheck]
+      taskRef:
```

Comment:
We only need 1 `taskRef` per task, this one is missing the task name, the one below in L114 has the name but not the other params.